Arrowgene.Ddon.WebServer/AccountRoute.cs

```diff
@@ -33,6 +34,53 @@ private class AccountResponse
             public string Token { get; set; }
         }
 
+        private class AccountVerification
+        {
+
+            public bool Error { get; set; }
+            public string Message { get; set; }
+            public string AccountID { get; set; }
```

Comment:
Account IDs are a different concept used in the codebase, if in this class it's used to mean usernames, please rename the variable to username, especially in user facing errors.

Reply:
Renamed :)